src/gtk/menu.cpp

```diff
@@ -678,6 +679,35 @@ void wxMenuItem::SetGtkLabel()
 #endif // wxUSE_ACCEL
 }
 
+#if wxUSE_ACCEL
+void wxMenuItem::SetGtkExtraAccels()
+{
+    for (wxString extra_accel : m_extra_accels)
+    {
+        wxAcceleratorEntry entry;
+        if ( entry.FromString(extra_accel) )
+        {
+            const wxString extra_hot_key = GetGtkHotKeyFromAccel(&entry);
+            if ( !extra_hot_key.empty() )
+            {
+                guint extra_accel_key;
+                GdkModifierType extra_accel_mods;
+                gtk_accelerator_parse(
+                    wxGTK_CONV_SYS(extra_hot_key),
+                    &extra_accel_key, &extra_accel_mods
+                );
+                if ( gtk_accelerator_valid(extra_accel_key, extra_accel_mods) )
```

Comment:
We need to log a message similar to the one in `wxGetGtkAccel()` (maybe it can refactored and reused?) here to avoid the situation in which accelerators are simply silently ignored and people don't know why they don't work.